src/YQReplacePoint.cc

```diff
@@ -44,8 +44,16 @@ void YQReplacePoint::showChild()
 
     if ( child )
     {
-	QWidget * qChild = (QWidget *) child->widgetRep();
-	qChild->show();
+        QWidget * qChild = (QWidget *) child->widgetRep();
```

Comment:
in general fix looks ok, but please write here comment why such complex stuff is needed to be done as it is not obvious. You do not need to comment what you do, only why it is needed to be done this way.